src/OrchardCore.Modules/OrchardCore.Media/Services/ChunkFileUploadService.cs

```diff
@@ -72,10 +72,10 @@ contentRangeHeader.Count is 0 ||
             : await chunkAsync(uploadId, formFile, contentRange);
     }
 
-    private async Task<IActionResult> CompleteUploadAsync(
```

Comment:
Why?!! 

You can mock the action results

Reply:
Because action results are on different layer, should not be in service layer